### PR TITLE
Remove redux from `example` package

### PR DESCRIPTION
**Pull request by TheZoker**
Current status:

| Example | Working | Broken before | Comment |
|---|:---:|:---:|---|
| 1169 | ✅ |  |  |
| 1220 | ✅ |  | No action buttons (and logic) |
| 1273 | ✅ |  |  |
| 1713 | ✅ |  |  |
| allOf | ✅ |  |  |
| anyOf | ✅ |  |  |
| anyOf_simple | ✅ |  |  |
| oneOf | ✅ |  |  |
| 1265_array | ✅ | ✅ |  |
| 1265_object | ✅ |  |  |
| 1265_simple | ✅ |  |  |
| 1273_simple | ✅ |  |  |
| oneOfArray | ✅ |  |  |
| anyOf-oneOf-allOf-resolve | ✅ | ✅ |  |
| array | ✅ |  | No action buttons (and logic) |
| nestedArray | ✅ | ✅ | No action buttons (and logic) |
| person | ✅ |  |  |
| array-with-detail | ✅ |  |  |
| array-with-detail-and-rule | ✅ |  |  |
| array-with-custom-element-label | ✅ |  |  |
| stringArray | ✅ |  |  |
| categorization | ✅ |  |  |
| categorizationstepper | ✅ |  |  |
| categorizationsteppernavbuttons | ✅ |  |  |
| control-options | ✅ |  |  |
| day1 | ✅ |  |  |
| day2 | ✅ |  |  |
| day3 | ✅ |  |  |
| day4 | ✅ |  |  |
| day5 | ✅ |  |  |
| day6 | ❌ | ✅ | Lot of code, for small feature |
| dates | ✅ |  |  |
| dynamic | ✅ |  |  |
| generate | ✅ |  |  |
| generate-ui | ✅ |  |  |
| layout-vertical | ✅ |  |  |
| layout-horizontal | ✅ |  |  |
| layout-group | ✅ |  |  |
| layout-complex | ✅ |  |  |
| rule | ✅ |  |  |
| rule-enable | ✅ |  | No action buttons (and logic) |
| resolve | ❌ |  | Removed in https://github.com/eclipsesource/jsonforms/pull/1829 |
| configDefault | ✅ |  |  |
| configCustom | ✅ |  |  |
| text | ✅ |  |  |
| numbers | ✅ |  |  |
| scope | ✅ |  |  |
| list-with-detail | ✅ | ✅ |  |
| list-with-detail-no-labelref | ✅ | ✅ |  |
| list-with-detail-registered | ✅ | ✅ |  |
| rootObject | ✅ |  |  |
| object | ✅ |  |  |
| i18n | ❌ |  | Lot of code, for small feature |
| issue-1253 | ✅ |  |  |
| issue-1253-wrong-path-binding | ✅ |  |  |
| issue-1253-cell-missing-cell | ✅ | ✅ | |
| issue-1253-add-button-empty-row | ✅ | ✅ | |
| issue-1253-enum-error | ✅ | ✅ |  |
| issue-1254 | ✅ |  |  |
| oneOf-recursive | ✅ |  |  |
| huge | ✅ | ✅ |  |
| default | ✅ |  |  |
| onChange | ✅ |  |  |
| enum | ✅ |  | No action buttons (and logic) |
| radio-group | ✅ |  |  |
| booleanToggle | ✅ |  |  |
| multi-enum | ✅ |  |  |
| enumInArray | ✅ |  |  |
| Readonly Fields | ✅ |  |  |
| 1779-string | ❌ | ✅ | No applicable renderer found (ListWithDetails works for objects, but not for strings/numbers) |
| 1779-number | ❌ | ✅ | No applicable renderer found (ListWithDetails works for objects, but not for strings/numbers) |
| ifthenelse | ✅ |  |  |

### CODE REVIEW

File: packages/examples/src/example.ts [hunk 1]
```diff
@@ -22,25 +22,31 @@
...
 
 export interface ExampleDescription {
   name: string;
   label: string;
   data: any;
   schema: JsonSchema;
   uischema: UISchemaElement;
+  uischemas?: JsonFormsUISchemaRegistryEntry[];
   config?: any;
+  actions?: any;
```
Comment (LukasBoll) [inline packages/examples/src/example.ts:41]: I think we can add these types here.
```suggestion
  actions?: {label: string, apply: (props: StateProps) => any}[];
```

File: packages/example/src/App.tsx [hunk 2]
```diff
@@ -65,16 +126,24 @@ class App extends Component<AppProps> {
...
           <h4 className='data-title'>Bound data</h4>
           <div className='data-content'>
-            <pre style={preStyle}>{this.props.dataAsString}</pre>
+            <pre>{dataAsString}</pre>
           </div>
           <div className='demoform'>
-            {this.props.getExtensionComponent()}
-            <JsonFormsDispatch onChange={this.props.onChange} />
+            <div className="buttons">
+              {actions?.map((buttons: any, index: number) => (
```
Comment (LukasBoll) [inline packages/example/src/App.tsx:133]: I think this should be `button`

File: packages/examples/src/examples/ruleInheritance.ts [hunk 3]
```diff
@@ -93,12 +94,36 @@ export const data = {
   toggleControl: true
 };
 
+const actions = [
+  [
```
Comment (LukasBoll) [inline packages/examples/src/examples/ruleInheritance.ts:98]: Currently the actions of ruleInheritance aren’t working. I think you added one pair of braces to many.
```suggestion
```

**Review by LukasBoll — CHANGES_REQUESTED** [review]
Overall Code looks good and the examples themselves are behaving well, but there are some problems when changing between examples, e.g.:
Changing from “oneOf (in Array)” to “array”: No appliciable renderer is found.
Changing from “Date” to ”DynamicChange”: The old Schema from Date is used.

**Comment by TheZoker** [comment]
Thanks for the review! Yes you are right, there seems to be an issue when switching examples. I'll have to investigate this further

**Review by LukasBoll — APPROVED** [review]
Looks good to me, thank you!

**Review by sdirix — APPROVED** [review]
LGTM! Thanks!

**Pull request merged**